libsession/src/main/java/org/session/libsession/messaging/jobs/GroupAvatarDownloadJob.kt

```diff
@@ -0,0 +1,51 @@
+package org.session.libsession.messaging.jobs
+
+import org.session.libsession.messaging.MessagingModuleConfiguration
+import org.session.libsession.messaging.open_groups.OpenGroupAPIV2
+import org.session.libsession.messaging.utilities.Data
+import org.session.libsession.utilities.GroupUtil
+
+class GroupAvatarDownloadJob(val room: String, val server: String) : Job {
+
+    override var delegate: JobDelegate? = null
+    override var id: String? = "$server.$room"
```

Comment:
Only concern I have with overriding the ID here is that it may be overwritten in the `JobQueue`'s `addWithoutExecuting` and, but we're using the values in the serialized data so it shouldn't be an issue here

Reply:
Oh I see, I missed that one place where the field is currently being overridden